packages/styles/scss/patterns/sub-patterns/card-link/README.md

```diff
@@ -21,4 +21,5 @@ Import the package css into the top of your main CSS file.
 | `.bx--card-link__title`   | Title element within the CardLink component |
 | `.bx--card-link__content` | Paragraph of text styling for CardLink      |
 | `.bx--card-link__footer`  | Footer aligned bottom with icon             |
-| `.bx--card-link__image`   | Footer aligned bottom with icon             |
+| `.bx--card-link__img`     | Footer aligned bottom with icon             |
```

Comment:
Maybe we could give this an accurate description before merging?

Reply:
oh whoops forgot to edit the desc, done!